packages/react-components/src/components/IncomingCallNotification.tsx

```diff
@@ -118,69 +141,105 @@ export interface IncomingCallNotificationProps {
  * @beta
  */
 export const IncomingCallNotification = (props: IncomingCallNotificationProps): JSX.Element => {
-  const { callerName, alertText, avatarImage, onAcceptWithAudio, onAcceptWithVideo, onReject, personaSize, styles } =
-    props;
+  const {
+    callerName,
+    alertText,
+    avatarImage,
+    onAcceptWithAudio,
+    onAcceptWithVideo,
+    onReject,
+    onDismiss,
+    personaSize,
+    styles,
+    strings
+  } = props;
   const theme = useTheme();
   /* @conditional-compile-remove(one-to-n-calling) */
   const localeStrings = useLocale().strings.IncomingCallNotification;
+  /* @conditional-compile-remove(one-to-n-calling) */
+  const formattedMessageString =
+    localeStrings.incomingCallNotificationPlaceholderAlert && callerName
+      ? _formatString(localeStrings.incomingCallNotificationPlaceholderAlert, { callerName: callerName })
+      : callerName;
   return (
     <Stack
-      horizontal
       tokens={{ childrenGap: '0.5rem' }}
       verticalAlign="center"
       styles={styles?.root ? styles.root : incomingCallToastStyle(theme)}
     >
-      <Stack
-        horizontalAlign="start"
-        styles={styles?.avatarContainer ? styles.avatarContainer : incomingCallToastAvatarContainerStyle}
-      >
-        <Persona
-          imageUrl={avatarImage}
-          text={callerName}
-          size={PersonaSize.size40}
-          coinSize={personaSize}
-          hidePersonaDetails={true}
-          aria-label={callerName}
-        />
-      </Stack>
-
-      <Stack grow={1} horizontalAlign="center" style={{ alignItems: 'flex-start', fontFamily: 'Segoe UI' }}>
-        <Stack style={{ fontSize: '0.875rem' }}>
-          <Text>
-            {callerName ??
-              /* @conditional-compile-remove(one-to-n-calling) */ localeStrings.incomingCallNotificationPlaceholderId}
-          </Text>
+      <Stack horizontal verticalAlign="center" tokens={{ childrenGap: personaSize ? '0.5rem' : '0rem' }}>
+        <Stack
+          horizontalAlign="start"
+          styles={styles?.avatarContainer ? styles.avatarContainer : incomingCallToastAvatarContainerStyle}
+        >
+          <Persona
+            imageUrl={avatarImage}
+            text={callerName}
+            size={PersonaSize.size24}
+            coinSize={personaSize}
+            hidePersonaDetails={true}
+            aria-label={callerName}
+          />
         </Stack>
-        <Stack style={{ fontSize: '0.75rem' }}>
-          <Text>
-            {alertText ??
-              /* @conditional-compile-remove(one-to-n-calling) */ localeStrings.incomingCallNotificationPlaceholderAlert}
-          </Text>
+
+        <Stack grow={1} horizontalAlign="center" style={{ alignItems: 'flex-start', fontFamily: 'Segoe UI' }}>
```

Comment:
```suggestion
        <Stack grow horizontalAlign="center" style={{ alignItems: 'flex-start', fontFamily: 'Segoe UI' }}>
```

Reply:
Also, if applying a Font family here, would it cause issues with contoso wanting to style font changes throughout?